### DISCUSSION TITLE
Is giving unnecessary usernames mandatory to get sponsored?

### DISCUSSION THREAD

**Discussion by dhruvkrishnavaid** [community/community#21486] [Other Feature Feedback, Questions, & Ideas]
<p><a class="mention" href="/SethClydesdale">@SethClydesdale</a> <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a><br>
This is the default format of setting up FUNDING.yml</p>
<pre><code class="lang-auto"># These are supported funding model platforms

github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
patreon: # Replace with a single Patreon username
open_collective: # Replace with a single Open Collective username
ko_fi: # Replace with a single Ko-fi username
tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
liberapay: # Replace with a single Liberapay username
issuehunt: # Replace with a single IssueHunt username
otechie: # Replace with a single Otechie username
custom: # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
</code></pre>
<p>My question is that - Is it necessary to give all these usernames to get sponsored?<br>
Like, can’t we give just 4 Github sponsor - enabled usernames?</p>
<pre><code class="lang-auto"># These are supported funding model platforms

github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
</code></pre>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/dhruvkrishnavaid">@dhruvkrishnavaid</a>,</p>
<p>No it is not necessary. The additional sponsor links are optional and are simply there for those who wish to use those specific funding platforms. You’re free to include whichever platform(s) you use for funding. For example, I only include a few custom links, such as one to my paypal or repo-specific donation page.</p>
<p>While it is not necessary to include all of these funding platforms, giving potential sponsors more options does increase the likelihood of you receiving funding, but don’t let that discourage you from keeping things simple if you wish to. It’s perfectly okay to include only your sponsor-enabled username. :+1:</p>
